docs/next-docs/stack.md

```diff
@@ -0,0 +1,38 @@
+  - [NextJS Tutorial](https://nextjs.org/learn/foundations/about-nextjs)
+  - [NextJS Docs](https://nextjs.org/docs)
+- [React](https://reactjs.org/) - A JavaScript library for building component-based user interfaces
+- [Typescript](https://www.typescriptlang.org/) - TypeScript is a strongly typed programming language that builds on JavaScript
+- [Chakra UI](https://chakra-ui.com/) - A UI library (Migration in progress)
+- [Algolia](https://www.algolia.com/) - Site indexing, rapid intra-site search results, and search analytics. [Learn more on how we implement Algolia for site search](./docs/ALGOLIA_DOCSEARCH.md).
+  - Primary implementation: `/src/components/Search/index.ts`
+- [Crowdin](https://crowdin.com/) - crowdsourcing for our translation efforts (See "Translation initiative" below)
+- [GitHub Actions](https://github.com/features/actions) - Manages CI/CD, and issue tracking
+- [Netlify](https://www.netlify.com/) - DNS management and primary host for `master` build.
+- [Storybook](https://storybook.js.org/) - For UI development, testing, and documentation. Check out [our storybook!](https://dev--63b7ea99632763723c7f4d6b.chromatic.com/)
+- [Chromatic](https://www.chromatic.com/) - Visual testing & UI reviews. Visit [our chromatic project](https://www.chromatic.com/builds?appId=63b7ea99632763723c7f4d6b)
+
+## Code structure
+
+| Folder                                 | Primary use                                                                                                                                                                                                                     |
+| -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `/src`                                 | Main source folder for development                                                                                                                                                                                              |
+| `/public/assets`                       | Image assets                                                                                                                                                                                                                    |
+| `/src/components`                      | React components that do not function as standalone pages                                                                                                                                                                       |
+| `/public/content`                      | Markdown/MDX files for site content stored here. <br>For example: `ethereum.org/en/about/` is built from `public/content/about/index.md` <br>The markdown files are parsed and rendered using `src/layouts/Static.tsx`\* layout |
```

Comment:
They aren't all rendered using the `Static` layout... maybe adjust the wording to say these md files are parsed by `[...slug].tsx`?